scripts/ci/gitlab/pipeline/check.yml

```diff
@@ -62,10 +62,10 @@ check-try-runtime:
     - |
       export RUST_LOG=remote-ext=debug,runtime=debug
       echo "---------- Running try-runtime for ${NETWORK} ----------"
-      time cargo build --release --locked -p "$NETWORK"-runtime
-      time cargo run --locked --release --features try-runtime try-runtime \
+      time cargo install --git https://github.com/paritytech/try-runtime-cli --branch liam-v0.1.0
```

Comment:
Why do we need to use this special branch here?

Reply:
the branch hasn't merged yet. I have a TODO in the PR description to remove the `--branch` before we merge